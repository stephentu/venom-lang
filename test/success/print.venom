x = "hello world";
y = 2 + 2;
z = 2.123 + 5;
a = 7 % 4;
b = a > 5;
c = 0.0;

print (x);
print (y);
print (z);
print (a);
print (b);
print (c);
