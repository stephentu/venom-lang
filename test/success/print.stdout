hello world
4
7.123
3
False
0.0
